src/encoded/types/suspension.py

```diff
@@ -56,123 +37,13 @@ class Suspension(Item):
     item_type = 'suspension'
     schema = load_schema('encoded:schemas/suspension.json')
     embedded = [
-        'biosample_ontology'
+        'biosample_ontology',
+        'donors',
+        'donors.diseases',
+        'donors.ethnicity'
     ]
 
 
-    @calculated_property(define=True,
-                         schema={"title": "Organisms",
-                                 "description": "The species the sample came from.",
-                                 "comment": "Do not submit. This is a calculated property",
-                                 "type": "array",
-                                 "items": {
-                                    "type": "string"
-                                    }
-                                 })
-    def organisms(self, request, registry, donors=None):
-        if donors is not None:
-            org_list = []
-            organisms = caclulate_donor_prop(self, request, donors, 'organism')
-            for organism in organisms:
-                organismObject = request.embed(organism, '@@object')
-                if organismObject.get('scientific_name') not in org_list:
-                    org_list.append(organismObject.get('scientific_name'))
-            return org_list
-
-
-    @calculated_property(define=True,
-                         schema={"title": "Sex",
-                                 "description": "The sex of the donor(s) that the sample came from.",
-                                 "comment": "Do not submit. This is a calculated property",
-                                 "type": "string",
-                                 "enum": [
-                                    "male",
-                                    "female",
-                                    "mixed",
-                                    "unknown"
-                                    ]
-                                 })
-    def sex(self, request, donors=None, organism=None):
```

Comment:
Removing this is going to break the facet above
I would update that to donors.sex (and change the title to 'Donor sex')